examples/solids/index.rst

```diff
@@ -338,16 +338,35 @@ Integrating by parts, we arrive at the weak form:
 where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary
 condition written in terms of the reference configuration.
 
-For simplicity, we express here the constitutive law in indicial notation:
+Equation :math:numref:`1st2nd` devises a method to model a hyperelastic solid with a Neo-Hookean constitutive law. Evaluatinig the derivative of this model, yields a forth order tensor:
```

Comment:
That equation is just a definition, `P = F S`. It isn't a method. It also isn't clear in this context why you are computing this derivative.